lib/rubocop/cop/lint/useless_else_without_rescue.rb

```diff
@@ -25,8 +25,12 @@ module Lint
       #     do_something_else
       #   end
       class UselessElseWithoutRescue < Base
+        extend TargetRubyVersion
+
         MSG = '`else` without `rescue` is useless.'
 
+        maximum_target_ruby_version 2.5
```

Comment:
The comment on line 8 is probably not necessary since this will add a ruby version note to the docs.

Reply:
They all have comments similar to this. I think those are fine to leave as rationale why the cop is now irrelevant.

Reply:
True, fair enough.